src/lib/battery/battery.cpp

```diff
@@ -223,17 +223,17 @@ Battery::determineWarning(bool connected)
 void
 Battery::computeScale()
 {
-	const float voltage_range = (_v_charged.get() - _v_empty.get());
+        //Using the filtered load voltage to give the most accurate reading.
+        //Checking validity of _voltage_filtered_v
+        if(_voltage_filtered_v > 2.1f) {
```

Comment:
Instead of a hardcoded magic number (2.1) can we do this based on existing parameters?

Reply:
Since the point of "2.1" was just to avoid division by 0, changed it to 0.1f and added comment.